src/main/scala/KinesisHandler.scala

```diff
@@ -5,10 +5,10 @@ import com.amazonaws.services.kinesis.AmazonKinesisClientBuilder
 import com.amazonaws.services.kinesis.model.{PutRecordRequest, Record}
 import org.apache.logging.log4j.scala.Logging
 
-import scala.concurrent.Future
-import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{ExecutionContext, Future}
 
 object KinesisHandler extends CrossAccount with Logging {
```

Comment:
can this file be deleted now?

Reply:
yes, i felt there was no problem leaving it in and not executed as it shouldn't make a big difference to the size of the jar but i suppose it could be deleted.  I am a don't-throw-things-away-unless-absolutely-necessary type of person :)